docs/using/administering/debugging.rst

```diff
@@ -33,6 +37,78 @@ It is possible to see the available unit names, and then view the logs with ``jo
    [root@node1]# journalctl -u flocker-container-agent
    [root@node1]# journalctl -u flocker-control
 
+Bug Reporting
+-------------
+
+When reporting issues with Flocker please include:
+
+* Flocker version,
+* operating system and version,
+* Linux kernel version,
+* Docker version and configuration,
+* node IP addresses,
+* node hostname,
+* recent syslog content,
+* and separate Flocker log files.
```

Comment:
It might make this a little easier to read by using complete sentences in the bulleted list:

* The version of Flocker you are using.
* Your operating system and version.
* Your Linux kernel version.
* The version of Docker you are using, and your configuration details.
* Your node IP addresses.
* Your node hostname.
* All recent syslog content.
* Any separate Flocker log files.